man/man8/facter.8

```diff
@@ -27,28 +27,32 @@ Collect and display facts about the current system\. The library behind Facter i
 If no queries are given, then all facts will be returned\.
 .
 .P
-Many of the command line options can also be set via the HOCON config file. This file can also be used to block the collection of certain fact groups\.
+Many of the command line options can also be set via the HOCON config file. This file can also be used to block or cache certain fact groups\.
 .
 .SH "OPTIONS"
 .
 .nf
-      \fB\-\-color\fR                      Enables color output\.
-      \fB\-\-config\fR                     A custom location for the chnfig file\.
+      \fB\-\-color\fR                      Enable color output\.
+      \fB\-\-config\fR                     A custom location for the config file\.
       \fB\-\-custom-dir\fR arg             A directory to use for custom facts\.
 \fB\-d, [ \-\-debug ]\fR                    Enable debug output\.
       \fB\-\-external-dir\fR arg           A directory to use for external facts\.
       \fB\-\-help\fR                       Print help and usage information\.
 \fB\-j, [ \-\-json ]\fR                     Output facts in JSON format\.
+      \fB\-\-list\-block\-groups\fR          List the names of all blockable fact groups\.
```

Comment:
Yes, I had tried it at least on Mac, and actually fixed some spacing that was bad before.